app/assets/stylesheets/application.scss

```diff
@@ -77,9 +77,11 @@ $primary-color: #008cba;
     background: rgba(0, 0, 0, 0.6);
     color: $white;
     font: bold 12px/24px Helvetica, Sans-Serif;
-    padding: 5px;
+    padding: 1px;
```

Comment:
Properties should be ordered background, color, font, margin-bottom, margin-top, padding, padding-left, padding-right